### ISSUE TITLE
Feature Request: Create a shortcut to search requests without needing to access sidebar

### ISSUE THREAD

**Issue by govindrai** [a85/POSTMan-Chrome-Extension#4527]
Feature:
I want to easily be able to fuzzy search a request by hitting, say, Command + F, and either have it open the side search bar and place the cursor so I can just start typing, OR,

better yet, think of Command+P in Sublime, where you can fuzzy search any file name, or slack's Command + T, or mac's spotlight (Command + Space), where a search box appears in the middle of the UI and you can just search. That would be epic, as now I could just keep my sidebar closed and gives me more room to work on my requests.

Regardless, I would say the Command + F functionality is not so intuitive. Why? Depending on which part of the postman UI is focused, invoking that command offers search on different parts of the UI. Therefore the command is too overloaded, imo. 

Basically, I'm just trying to easily search a request in my sidebar so give me a better shortcut/experience for that.

**Comment by dhineshwiz** [comment]
Hey @govindrai Thanks for suggestion, currently cmd+f can used for searching collection when collection sidebar is on focused. I am adding this as feature request

**Comment by aravind-arun** [comment]
Postman has a global search bar on the header. Ctrl + K is the shortcut that can be used from any part of Postman.  We also have filters in the universal search bar to filter the search results to show only requests.
